Aquality.Selenium.Template/Aquality.Selenium.Template/Aquality.Selenium.Template.csproj

```diff
@@ -16,14 +16,14 @@
 
   <ItemGroup>
     <PackageReference Include="Allure.Commons" Version="3.5.0.73" />
-    <PackageReference Include="Aquality.Selenium" Version="4.9.0" />
-    <PackageReference Include="AspectInjector" Version="2.8.0" />
+    <PackageReference Include="Aquality.Selenium" Version="4.9.1" />
+    <PackageReference Include="AspectInjector" Version="2.8.1" />
     <PackageReference Include="Humanizer" Version="2.14.1" />
     <PackageReference Include="NUnit.Allure" Version="1.2.1.1" />
     <PackageReference Include="AqualityTracking.Integrations.Core" Version="1.0.1" />
     <PackageReference Include="NUnit" Version="3.13.3" />
-    <PackageReference Include="RestSharp" Version="108.0.1" />
-    <PackageReference Include="System.Drawing.Common" Version="6.0.0" />
+    <PackageReference Include="RestSharp" Version="108.0.3" />
+    <PackageReference Include="System.Drawing.Common" Version="7.0.0" />
```

Comment:
please reverd to 6.0.0

Reply:
The version was returned in a subsequent commit - https://github.com/aquality-automation/aquality-selenium-dotnet-template/pull/14/commits/ebf288194f0d90fd32dd1f2343d48873e6ec34c4